From e8e9ff838d3a63b2bbafb62e8532d711ad9a2b78 Mon Sep 17 00:00:00 2001
From: Cedric Neveux <cedric.neveux@nxp.com>
Date: Fri, 23 Nov 2018 11:24:08 +0100
Subject: [PATCH 5191/5242] MLK-20450 Fix Linux NO SMP kernel - GPU

commit  e98406a16d9d2f91232dbc8470148f8c0d20fc89 from
https://source.codeaurora.org/external/imx/linux-imx.git

       - Fix GPU-viv drivers to build without SMP

Signed-off-by: Cedric Neveux <cedric.neveux@nxp.com>
(cherry picked from commit 6558c2881b0cd1441708933fa1ac397f54b5aa70)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
index bd3ea86..cb52a3b6 100644
--- a/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
+++ b/drivers/mxc/gpu-viv/hal/os/linux/kernel/gc_hal_kernel_os.c
@@ -5808,7 +5808,7 @@ gceSTATUS gckOS_UserLogicalToPhysical(
     )
 {
     gceSTATUS status;
-    gcsSIGNAL_PTR signal;
+    gcsSIGNAL_PTR signal = gcvNULL;
     int done;
     unsigned long flags;
 
-- 
1.7.9.5

